### DISCUSSION TITLE
Paredit in Cider REPL appears to prevent clojure repl evaluation

### DISCUSSION THREAD

**Discussion by RogerWWilliams** [clojure-emacs/cider#3334] [General]
Hello All:

I just recently found that the below entry in my init.el for paredit v20221127.1452 when using cider 20230413.743 prevents any repl evaluation once starting up a repl via cider-jack-in-clj.  Regardless of whether or not a project has been loaded.  Simply commenting out the below line fixed everything in my case.

(add-hook 'cider-repl-mode-hook #'paredit-mode)

Not sure how long this has been occurring as its been a while since I "jacked-in" to anything and I tend to update my Emacs packages every few weeks.

I added the above line to my init.el a while back after reading https://tbellisiv.gitbooks.io/clojure-emacs-cider-intro/content/Spiking_Cider/Paredit.html

Just pointing it out in case anyone else has hit this.

Thank you,
Roger

**Comment by bbatsov** [comment]
See https://docs.cider.mx/cider/troubleshooting.html#pressing-ret-in-the-repl-does-not-evaluate-forms That's basically the result of an unfortunate upstream change in Paredit.

**Reply by yuhan0** [reply]
Would it be a good idea to defensively override the Paredit binding in cider-repl-mode? It appears we can do that with `minor-mode-overriding-map-alist`.

**Reply by bbatsov** [reply]
Yeah, I think that'd be a good idea.

**Comment by RogerWWilliams** [comment]
Thank you for the link!  Sorry I didn't see this when I was working on it.
